src/infra/HttpServiceImpl.ts

```diff
@@ -22,10 +22,13 @@ export class HttpServiceImpl implements HttpService {
   get<T>(
     url: string,
     queryParams: QueryParams,
+    authToken?: string,
   ): Promise<T> {
     return this.fetch(url, queryParams, {
       method: RequestMethods.GET,
-      credentials: 'include'
+      credentials: 'include',
```

Comment:
we have credentials set to `include`. When we are passing authorization headers, then Access-Control-Allow-Origin in the response header can't be '*' or else it's block by CORS policy. Is this something backend will adjust?

Reply:
Are you seeing 'access-control-allow-origin' being set to '*'?  I'm seeing specific domains being set